src/components/card/Card.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+
+import {
+  useCollectionCoverArt,
+  useUserProfilePicture
+} from 'audius-client/src/common/hooks/useImageSize'
+import {
+  CoverArtSizes,
+  ProfilePictureSizes,
+  SquareSizes
+} from 'audius-client/src/common/models/ImageSizes'
+import { User } from 'audius-client/src/common/models/User'
+import {
+  StyleProp,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  ViewStyle
+} from 'react-native'
+
+import DynamicImage from 'app/components/dynamic-image'
+import UserBadges from 'app/components/user-badges/UserBadges'
+import { ThemeColors, useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ID } from 'app/store/notifications/types'
+import { font } from 'app/styles'
+
+export type CardType = 'user' | 'collection'
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    shadowContainer: {
+      elevation: 2,
```

Comment:
Let's update this to use `Shadow` from react-native-shadow-2 since we have it now! https://github.com/AudiusProject/audius-mobile-client/pull/232/files

Reply:
💯